final_task/pycalc/pycalc.py

```diff
@@ -0,0 +1,190 @@
+"""This is a main module with general functions
+
+:author: Fyodorova Elena
+"""
+from re import findall
+import pycalc.Data as Data
+import argparse
+import importlib
+
+
+def pycalc(string: str):
+    """Main function
+
+    :param string: input expression
+    :type string: str
+    :return: result of calculation or error message
+    """
+    brackets_list = findall('[(-)]', string)  # Searching for brackets to check a balance of them
+    if brackets_list.count('(') != brackets_list.count(')'):
```

Comment:
Regular expression above is unnecessery in this case. You can repace `brackets_list` with `string`, remove row above and it will work just right
```suggestion
    if string.count('(') != string.count(')'):
```